x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -219,7 +240,18 @@ const GraphOverlayComponent: React.FC<OwnProps> = ({ timelineId }) => {
     [defaultDataView.patternList, isInTimeline, timelinePatterns]
   );
 
-  if (fullScreen && !isInTimeline) {
+  if (!isInTimeline && sessionViewId !== null) {
```

Comment:
We don't need the `fullScreen` check here anymore?

Reply:
How it should behave in the fullScreen mode?

Reply:
like analyzer i think, which it does, we just have a small issue with the hard coded unless specified height of 500px coming from here https://github.com/elastic/kibana/blob/main/x-pack/plugins/session_view/public/components/session_view/styles.ts#L16 @zizhouW @mitodrummer @opauloh I think we should change the styles here so that the search bar/detail panel button take up as much space as the constituent components need, and then the process tree grows to fill the remaining part of a top level container. Passing a height will be buggy/hard for no reason with the full screen functionality in data grid/timeline I think. What do y'all think?

Reply:
so, the current way it works, it can't have auto height, because of infinite scrolling, so in order to enable full screen you would need to manually change the `height` you're passing to SessionView on FullScreen mode, something like this:

```js
sessionView.getSessionView({
   sessionEntityId,
   height: fullScreen ? 'calc(100vh - 118px)' : '500px'
}) 
```

but having to know the height of the SessionView's search bar (118px), and the default height of the session view (500px) isn't ideal, so we are willing to change later to something more like this

```js
sessionView.getSessionView({
   sessionEntityId,
   isFullScreen: fullScreen
}) 
```